mpa/modules/ov/models/mmcls/heads/cls_head.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from mmcls.models.builder import HEADS
+from mmcls.models.heads import ClsHead as OriginClsHead
+
+
+@HEADS.register_module()
+class ClsHead(OriginClsHead):
```

Comment:
This conflicts with the Head of the existing mmcls. we need to change the name
Or if there is no problem, we should `force=True`.

Reply:
Since this class merely adds negligible features, I believe `force=True` would be enough.
If you insisted to rename it, I would. Let me know.

Reply:
I also think it's best to do `force=True` if this class doesn't have a big impact on the existing ones.